### PR TITLE
Unraid Docker template installation

### PR DESCRIPTION
**Pull request by f0rc3d**
Easy installation method for unraid docker users.

### CODE REVIEW

File: my-prefetcharr.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  <Config Name="Sonarr URL" Target="SONARR_URL" Default="" Mode="" Description="http://example.com/sonarr" Type="Variable" Display="always" Required="true" Mask="false"></Config>
+  <Config Name="Sonarr API Key" Target="SONARR_API_KEY" Default="" Mode="" Description="API key for Sonarr" Type="Variable" Display="always" Required="true" Mask="true"></Config>
+  <Config Name="Logging Directory" Target="LOG_DIR" Default="/log" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false">/log</Config>
+  <Config Name="Log level" Target="RUST_LOG" Default="prefetcharr=debug" Mode="" Description="Log level" Type="Variable" Display="always" Required="true" Mask="false">prefetcharr=debug</Config>
+  <Config Name="Polling Interval" Target="INTERVAL" Default="900" Mode="" Description="Polling interval in seconds" Type="Variable" Display="always" Required="true" Mask="false">900</Config>
+  <Config Name="Episodes Remaining Trigger" Target="REMAINING_EPISODES" Default="2" Mode="" Description="The last  episodes trigger a search" Type="Variable" Display="always" Required="true" Mask="false">2</Config>
```
Comment (p-hueber) [inline my-prefetcharr.xml:32]: Can you fix the description on this one? You dropped the `<NUM>` here and now the sentence doesn't make much sense.

**Comment by p-hueber** [comment]
hi @f0rc3d, thank you for your contribution!
can you point me at some documentation for these templates for future maintenance?

File: README.md [hunk 2]
```diff
@@ -108,3 +108,8 @@ compose up -d prefetcharr`.
 Once the container is running, you may want to check the logs for errors. You
 can do so by either calling `docker logs prefetcharr` or by checking the logging
 directory you configured.
+
+### Unraid Docker installation
+
+Disable Docker in unraid. Upload my-prefetcharr.xml to /boot/config/plugins/dockerMan/templates-user. Enable Docker.
+Add new Docker image from template. (Docker> Add Container> Template> User templates> prefetcharr)
```
Comment (p-hueber) [inline README.md:115]: please keep it within a 80 column line width where possible and put file names and paths in \`\`.

File: my-prefetcharr.xml (hunk 1 above)
Comment (f0rc3d) [reply my-prefetcharr.xml]: Ah thank you, fixed

File: README.md (hunk 2 above)
Comment (f0rc3d) [reply README.md]: Should be better now

Comment (f0rc3d) [reply README.md]: Unraid has a little bit on templates here: https://docs.unraid.net/unraid-os/manual/docker-management/#templates
Here's a community guide: https://selfhosters.net/docker/templating/templating/

The fastest way to update this if unraid makes changes would be to make a new template in unraid and export it.

**Pull request merged**